packages/expo-dev-menu/ios/DevMenuDevOptionsDelegate.swift

```diff
@@ -24,7 +24,9 @@ class DevMenuDevOptionsDelegate {
     // No native splash screen registered for given view controller. Call 'SplashScreen.show' for given view controller first.
     DevMenuManager.shared.hideMenu()
 
-    bridge?.requestReload()
+    DispatchQueue.main.async {
+      RCTTriggerReloadCommandListeners("Dev menu - reload")
+    }
```

Comment:
`requestReload` is deprecated 
https://github.com/facebook/react-native/blob/main/packages/react-native/React/Base/RCTBridge.m#L325-L331